postgresql/resource_postgresql_grant.go

```diff
@@ -646,7 +665,41 @@ func createRevokeQuery(d *schema.ResourceData) string {
 				pq.QuoteIdentifier(d.Get("role").(string)),
 			)
 		}
-	case "TABLE", "SEQUENCE", "FUNCTION", "PROCEDURE", "ROUTINE":
+	case "FUNCTION", "PROCEDURE", "ROUTINE":
```

Comment:
Same as above, these types have a separate quoting style so had to be split out.